src/clientsettingsdlg.cpp

```diff
@@ -388,10 +388,10 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
     chbDetectFeedback->setAccessibleName ( tr ( "Feedback Protection check box" ) );
 
     // init driver button
```

Comment:
Is there any other code relating to the button that should have the same treatment?  It's a bit odd only to change the label or hide the button - we shouldn't even have the "click" handling code if the button is hidden.

Reply:
Guess hiding the button (when the button was introduced) was the easy thing to do without impacting/changing too much code and because the button is not available, you cannot trigger the click handling. Will check the code and update accordingly.

Reply:
Do you think these changes should be code for JACK on Mac as well? 
As in that case the code should use the "WITH_JACK" in the if block instead of "JACK_REPLACES_ASIO".

Reply:
I'm not familiar with the preprocessor defines created in the Jamulus.pro for these situations.

Reply:
Going to update and use WITH_JACK. This is defined on all builds (when JACK is used) and will included _WIN32 as the button is only applicable to Windows.